bin/index.js

```diff
@@ -83,7 +83,7 @@ var getNetworkAddress = () => {
         let file = '';
         if (fileName) {
             usageMessage = `Scan the QR-Code to access '${fileName}' file on your phone`;
-            file = '/' + fileName;
+            file = '/' + encodeURIComponent(fileName);
```

Comment:
@colgin 
great! but if you do this when we have the full `shareAddress` we can support Unicode directory names too.
you can do something like this after line 94: `shareAddress  = encodeURI(shareAddress)`

thank you for this cool PR

Reply:
Actually, Unicode directory is supported now. 
```bash
sharing ./测试
```
it works!

Moreover, browser will encode the url automantically when we click the a element.

So, I think it's unnecessary to encode the whole address.